pages/glossary/autoplay.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Autoplay
+key: Autoplay
+unambiguous: true
+objective: false
+---
+
+An `audio` or `video` element _autoplays_ if the included media starts playing without the user interacting with the element.

```

Comment:
1. The controls to activate don't need to be part of the video / audio element. A separate play button should also pass
2. We should make sure this doesn't trigger on scroll or navigation, so just "without the user interacting" isn't enough here.


```suggestion
An `audio` or `video` element _autoplays_ if the included media starts playing without the user activating a control.
```